2주차/11866/11866_Java_짱구.java

```diff
@@ -0,0 +1,39 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.LinkedList;
+import java.util.StringTokenizer;
+import java.util.stream.Collectors;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st = new StringTokenizer(br.readLine());
+
+        int N = Integer.parseInt(st.nextToken());
+        int K = Integer.parseInt(st.nextToken());
+
+        LinkedList<Integer> list = new LinkedList<>();
```

Comment:
코딩 테스트와는 관련 없지만 습관화하면 좋을 것 같아요😄
아니면 이렇게 사용하면 메모리 효율이 더 좋은가요 ?-?
```suggestion
        List<Integer> list = new LinkedList<>();
```